include/fastgltf/types.hpp

```diff
@@ -1309,6 +1309,11 @@ namespace fastgltf {
          */
         std::variant<TRS, TransformMatrix> transform;
 
+        /**
+         * Only ever non-empty when EXT_mesh_gpu_instancing is enabled and used by the asset.
+         */
+        pmr::SmallVector<std::pair<std::pmr::string, std::size_t>, 4> instancingAttributes;
```

Comment:
Would it make sense to add a function like the findAttribute function on primitives? Seems like they have the same signature and same basic concept.

Reply:
Definitely !

Reply:
And done (I've also added some `noexcept`)